cmake/FindRocksDB.cmake

```diff
@@ -4,5 +4,20 @@ find_path(ROCKSDB_INCLUDE_DIR
   NAMES rocksdb/db.h
   PATH_SUFFIXES include)
 
+if(ROCKSDB_INCLUDE_DIR AND EXISTS "${ROCKSDB_INCLUDE_DIR}/rocksdb/version.h")
+  foreach(ver "MAJOR" "MINOR" "PATCH")
+    file(STRINGS "${ROCKSDB_INCLUDE_DIR}/rocksdb/version.h" ROCKSDB_VER_${ver}_LINE
+      REGEX "^#define[ \t]+ROCKSDB_${ver}[ \t]+[0-9]+$")
+    string(REGEX REPLACE "^#define[ \t]+ROCKSDB_${ver}[ \t]+([0-9]+)$"
+      "\\1" ROCKSDB_VERSION_${ver} "${ROCKSDB_VER_${ver}_LINE}")
+    unset(${ROCKSDB_VER_${ver}_LINE})
+  endforeach()
+  set(ROCKSDB_VERSION_STRING
+    "${ROCKSDB_VERSION_MAJOR}.${ROCKSDB_VERSION_MINOR}.${ROCKSDB_VERSION_PATCH}")
+
+  message(STATUS "Found RocksDB version: ${ROCKSDB_VERSION_STRING}")
```

Comment:
@xumengpanda, In this file FindRocksDB.cmake, enforcing exact version match to 6.22.1. 
Here I am just printing a log message to display the version that exists on the machine before the cmake binds to 6.22.1 rocksdb version.